.github/workflows/build-ws-container.yml

```diff
@@ -8,14 +8,31 @@ on:
 
 jobs:
   build:
-    runs-on: ubuntu-latest
+    runs-on: ubuntu-24.04-arm
```

Comment:
If GitHub provides two possible runners, why don't not we split it up in different `runs-on` profiles so we can benefit from native and not emulated performance.

Reply:
At the moment, there is no easy way to merge an image built on two different machines into one tag. You cannot push a version of an image for a certain architecture into an existing tag in a registry, that would overwrite the entire image. If we built the image on two different machines, we would have to use two different tags like `cockpit/ws:337-amd64` and `cockpit/ws:337-arm64`. Those would be two separate images and not one multi-platform image, which is rather inconvenient, as you have to specify your machine’s architecture when using the image and therefore run into problems e.g. when sharing the same Docker Compose configuration across multiple machines with different architectures.

Reply:
> The tasks container is already built that way: https://github.com/cockpit-project/cockpituous/blob/main/.github/workflows/build-tasks.yml. The two arm64/x86 builds are then "merged" via a manifest. This seems much nicer?

Oh nice, with `podman manifest` it seems doable. There also is an experimental [`docker manifest`](https://docs.docker.com/reference/cli/docker/manifest/) command, will check whether it works the same and update the PR later today.

Reply:
I think we would prefer using `podman manifest` so we can easily test it locally.

Reply:
> I think we would prefer using `podman manifest` so we can easily test it locally.

I will definitely prioritize Podman, just wanted to check if an equivalent setup with Docker is possible so we can keep the current dual setup with the `$RUNC` variable – or is this no longer necessary at all?

Reply:
@martinpitt I think we can drop the docker based workflow and go full podman as we already do this for the tasks container?

Reply:
I faintly remember that we had to build with docker because podman built images were not compatible with at least older docker images. If that's not an issue any more, sure!

Reply:
> I faintly remember that we had to build with docker because podman built images were not compatible with at least older docker images. If that's not an issue any more, sure!

Just tested running `ghcr.io/marvinruder/cockpit`, built by Podman, on Debian Bullseye running Docker 20 (very old), no issues there.